docs/source/developer_guide/release.rst

```diff
@@ -6,13 +6,13 @@ Releases
 This section is for core developers. To make a new release, you need
 push-to-write access on our main branch.
 
-sktime is not a pure Python package and depends on some non-Python code
-including Cython and C. We distribute compiled files, called wheels, for
+`sktime` is distributed as the so-called wheels, for
```

Comment:
strange English.
"`sktime` is distributed in package builds, so-called wheels, for ..."
?

Reply:
The actual line is ``sktime` is distributed as the so-called wheels, for ...` is that strange?

Reply:
I think its ok

Reply:
well, it reads a bit nebulous to someone who does not know what wheels are.
So either assume a reference frame of someone knowing this (then there´s no need for "so-called"), or assume a frame where the reader doesn´t, and give an in-line definition of what wheels are.